### PR TITLE
Fix: unobtainable arcanums

### PR DESCRIPTION
**Pull request by baranzadeoglu**
the 3 items are changed in accordance with
https://tbc.wowhead.com/object=160845/dark-coffer

## Changes Proposed:
-  change quest required for 3 items to 0
-  change the drop rate and amount in accordance with 
https://tbc.wowhead.com/object=160845/dark-coffer

## Issues Addressed:

- Closes 
- https://github.com/chromiecraft/chromiecraft/issues/2218

## Tests Performed:

- not tested
- 

## How to Test the Changes:

1. Mysql query: 
`select * from gameobject_loot_template where gameobject_loot_template.Entry="11103" and  gameobject_loot_template.Chance > 25;`
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by IntelligentQuantum** [comment]
```
 Table 'gameobject_loot_template' Entry 11103 Item 11752: MaxCount (1) less that MinCount (2) - skipped
Table 'gameobject_loot_template' Entry 11103 Item 11751: MaxCount (1) less that MinCount (2) - skipped
```

**Comment by baranzadeoglu** [comment]
> ```
> Table 'gameobject_loot_template' Entry 11103 Item 11752: MaxCount (1) less that MinCount (2) - skipped
> Table 'gameobject_loot_template' Entry 11103 Item 11751: MaxCount (1) less that MinCount (2) - skipped
> ```

fixed.

**Review by IntelligentQuantum — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1635882858166720225.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1635882858166720225');
+
+DELETE FROM `gameobject_loot_template` WHERE (`Entry` = 11103) AND (`Item` IN (11751, 11752, 11753));
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(11103, 11751, 0, 46, 0, 1, 0, 1, 2, 'Dark Coffer - Burning Essence'),
+(11103, 11752, 0, 43, 0, 1, 0, 1, 2, 'Dark Coffer - Black Blood of the Tormented'),
+(11103, 11753, 0, 39, 0, 1, 0, 1, 1, 'Dark Coffer - Eye of Kajal');
+
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1635882858166720225.sql:8]: extra line here.

**Review by Nefertumm — APPROVED** [review]

**Comment by Nefertumm** [comment]
Although, seems referred to https://github.com/azerothcore/azerothcore-wotlk/pull/8895

**Comment by j-hdt** [comment]
merge date available?

**Comment by Hacki95** [comment]
> 
> 
> merge date available?

Once someone finds the time to test it.

**Comment by Nyeriah** [comment]
Tested, works as intended

**Pull request merged**